FlashCards.obitom67/FlashCards.obitom67/StudySession.cs

```diff
@@ -0,0 +1,93 @@
+﻿
+using System.Data.SqlClient;
+using Dapper;
+using Spectre.Console;
+
+
+namespace FlashCards.obitom67
+{
+    internal class StudySession
+    {
+
+        int StudyId { get; set; }
+        int StackId { get; set; }
+        int CorrectQ {  get; set; }
+        int TotalQ { get; set; }
+        string Date { get; set; }
+
+
+        public static void StartSession()
+        {
+            StudySession studySes = new StudySession();
+            string connectionString = ConfigurationManager.AppSettings.Get("key1");
+            var currentStack = Stack.DisplayStacks();
+            string answer;
+            using (var connection = new SqlConnection(connectionString))
+            {
+
+                string flashcardsSql = $"SELECT * FROM dbo.Flashcard WHERE StackId = {currentStack.StackId}";
```

Comment:
Next time, please use parameterised sql commands. Otherwise your program will be vulnerable to SQL Injection.